src/main/java/com/programmers/springbootbasic/domain/voucher/infrastructure/FilePersistenceVoucherRepository.java

```diff
@@ -0,0 +1,47 @@
+package com.programmers.springbootbasic.domain.voucher.infrastructure;
+
+import static com.programmers.springbootbasic.exception.ErrorCode.INVALID_FILE_PATH;
+
+import com.programmers.springbootbasic.domain.voucher.domain.VoucherRepository;
+import com.programmers.springbootbasic.domain.voucher.domain.entity.Voucher;
+import com.programmers.springbootbasic.exception.exceptionClass.VoucherException;
+import com.programmers.springbootbasic.util.FileManager;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@Profile("dev")
+@Primary
+public class FilePersistenceVoucherRepository implements VoucherRepository {
+
+    private final List<FileManager> fileManagerList;
+    private FileManager fileManager;
+    private String fileName;
```

Comment:
```suggestion
    private final String fileName;
```

생성자 인자 값으로 받아서 초기화 하는 것이기 때문에 final 키워드로 정의하는 것이 좋겠네요.!

Reply:
[dd13894](https://github.com/prgrms-be-devcourse/springboot-basic/pull/902/commits/dd138943e77d7dbf5aff52cbe0565d5aa80afdc1)

변경하였습니다!